src/script/auth/AuthRepository.ts

```diff
@@ -49,12 +45,6 @@ export class AuthRepository {
     this.logger = getLogger('AuthRepository');
   }
 
-  init(): Promise<Context> {
```

Comment:
This is actually included in the `core.init` call in the `app.ts` file. 
This was redundant